cmd/minikube/cmd/root.go

```diff
@@ -34,7 +34,10 @@ import (
 var dirs = [...]string{
 	constants.Minipath,
 	constants.MakeMiniPath("certs"),
-	constants.MakeMiniPath("machines")}
+	constants.MakeMiniPath("machines"),
+	constants.MakeMiniPath("iso"),
```

Comment:
Would you mind sticking iso and localkube into a top-level directory called cache?